model_compression_toolkit/hardware_models/keras_hardware_model/keras_default.py

```diff
@@ -0,0 +1,73 @@
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+import tensorflow as tf
+from keras.layers import Conv2D, Dense, Reshape, Flatten, Cropping2D, ZeroPadding2D, \
+    Dropout, MaxPooling2D, ReLU, Activation, Add, PReLU, BatchNormalization, Conv2DTranspose, DepthwiseConv2D
+
+
+from model_compression_toolkit.hardware_models.default_hwm import get_default_hardware_model
+
+import model_compression_toolkit as mct
+hwm = mct.hardware_representation
+
+
+def get_default_hwm_keras():
+    default_hwm = get_default_hardware_model()
+    default_hwm_keras = hwm.FrameworkHardwareModel(default_hwm,
+                                                   name='default_hwm_keras')
+
+    with default_hwm_keras:
+        hwm.OperationsSetToLayers("NoQuantization", [Reshape,
+                                                     tf.reshape,
+                                                     Flatten,
+                                                     Cropping2D,
+                                                     ZeroPadding2D,
+                                                     Dropout,
+                                                     MaxPooling2D,
+                                                     tf.split,
+                                                     tf.quantization.fake_quant_with_min_max_vars,
+                                                     BatchNormalization])
+
+        hwm.OperationsSetToLayers("Conv", [Conv2D,
+                                           DepthwiseConv2D])
+
+        hwm.OperationsSetToLayers("FullyConnected", [Dense])
+
+        hwm.OperationsSetToLayers("ConvTranspose", [Conv2DTranspose])
+
+        hwm.OperationsSetToLayers("AnyReLU", [tf.nn.relu,
+                                              tf.nn.relu6,
+                                              hwm.LayerFilterParams(ReLU, negative_slope=0.0),
+                                              hwm.LayerFilterParams(Activation, activation="relu")])
+
+        hwm.OperationsSetToLayers("Add", [tf.add,
+                                          Add])
+
+        hwm.OperationsSetToLayers("PReLU", [PReLU])
+
+        hwm.OperationsSetToLayers("Swish", [tf.nn.swish,
+                                            hwm.LayerFilterParams(Activation, activation="swish")])
+
+        hwm.OperationsSetToLayers("Sigmoid", [tf.nn.sigmoid,
+                                              hwm.LayerFilterParams(Activation, activation="sigmoid")])
+
+        hwm.OperationsSetToLayers("Tanh", [tf.nn.tanh,
+                                           hwm.LayerFilterParams(Activation, activation="tanh")])
+
+    return default_hwm_keras
+
+
+KERAS_DEFAULT_MODEL = get_default_hwm_keras()
```

Comment:
Why do we need this constant? relevant to all models

Reply:
fixed